CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Add new flavour with workflows and makefile for cluster app
```

Comment:
What is a cluster app? When would devs use this makefile or workflow

Reply:
With a cluster apps, I mean repositories like `cluster-aws`, `cluster-azure`, etc.

A similar version of the Makefile and workflows of this PR is already in use in [cluster-azure](https://github.com/giantswarm/cluster-azure).
You can take a look here:
[Makefile](https://github.com/giantswarm/cluster-azure/blob/main/Makefile.schema.mk)
[workflow](https://github.com/giantswarm/cluster-azure/blob/main/.github/workflows/json_schema.yaml)

Also for a explanation why we differentiate between a cluster-app and a regular app take a look at [this comment](https://github.com/giantswarm/devctl/pull/528#issuecomment-1459876880).

Reply:
thanks for the explanation. I was not aware of the term.